GANDLF/utils/gandlf_logger.py

```diff
@@ -0,0 +1,61 @@
+import logging
+import yaml
+from pathlib import Path
+from importlib import resources
+import tempfile
+from GANDLF.utils import get_unique_timestamp
+
+
+def _create_tmp_log_file():
+    tmp_dir = Path(tempfile.gettempdir())
+    log_dir = Path.joinpath(tmp_dir, ".gandlf")
+    log_dir.mkdir(parents=True, exist_ok=True)
+    log_file = Path.joinpath(log_dir, get_unique_timestamp() + ".log")
+    return log_file
+
+
+def _create_log_file(log_file):
+    log_file = Path(log_file)
+    log_file.write_text("Starting GaNDLF logging session \n")
+
+
+def _save_logs_in_file(log_file, config_path):
+    with resources.open_text("GANDLF", config_path) as file:
+        config_dict = yaml.safe_load(file)
+        config_dict["handlers"]["rotatingFileHandler"]["filename"] = str(log_file)
+        logging.config.dictConfig(config_dict)
+
+
+def gandlf_logger_setup(log_file=None, config_path="logging_config.yaml") -> None:
```

Comment:
Just a minor semantic change (since it is already part of the top-level `GANDLF` module). I am guessing this would have some other associated changes.

```suggestion
def logger_setup(log_file=None, config_path="logging_config.yaml") -> None:
```

Reply:
We can change it